_docs_integrate/delete-identities.md

```diff
@@ -0,0 +1,113 @@
+---
+# Start automatic generation
+permalink: integrate/delete-identities
+redirect_from:
+  - /integrate/delete-identity-from-enmeshed
+published: true
+title: "Delete Identities"
+type: scenario
+toc: true
+properties:
+  - id: SC061
+  - category: Identities and Relationships
+  - description:
+  - customer: All
+  - component: integrate
+  - level: Beginner
+  - implementation status: DONE
+  - documentation status: DONE
+  - published: true
+  - link: delete-identities
+require:
+required_by:
+  - integrate/exchange-messages
+# End automatic generation
+---
+
+It is not yet possible to utilize the use cases regarding the [IdentityDeletionProcesses]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess) mentioned below as an Integrator of a Connector. Instead, these functionalities are currently only offered to App users in order to be able to delete their [Identity]({% link _docs_integrate/data-model-overview.md %}#identity). Even if some use cases still have to be provided to the Integrators of Connectors for deleting their Identity, they benefit from the description of the process of Identity deletion given in this guide. Indeed, if an App user with whom a Connector has established a Relationship deletes their Identity, it is affected by [side effects on this Relationship]({% link _docs_integrate/delete-identities.md %}#side-effects-of-identity-deletion-on-relationships).
+{: .notice--warning}
+
+Regardless of whether an [Identity]({% link _docs_integrate/data-model-overview.md %}#identity) is an App user or a Connector, it is stored on the Backbone.
+This guide describes the different [options for Identity deletion]({% link _docs_integrate/delete-identities.md %}#options-for-identity-deletion) from the Backbone.
```

Comment:
Just FYI: If you want to link to sections on the same page, it is sufficient to use `[text](#section-title)`. That doesn't mean that you have to change it, but it might simplify your work in the future.